bang/templates/include/costumeTopIllustration.html

```diff
@@ -178,4 +179,11 @@
         {% endif %}
     }
 </script>
+{% else %}
+<script>
+    function loadModelViewerAjax() {
+        // well, this is awkward...
+    }
```

Comment:
So what does the item view look like for costumes that don't have a live2d? Just the fields? Do you get any JS error?

Reply:
yep, just fields. the stub `loadModelViewerAjax` is here to avoid a JS error from the ajax callback not existing.